gs.h

```diff
@@ -8126,6 +8127,62 @@ gs_camera_screen_to_world(const gs_camera_t* cam, gs_vec3 coords, s32 view_x, s3
     return wc;
 }
 
+gs_inline gs_mat4 
+gs_mat4_sky_look_at(gs_vec3 position, gs_vec3 target, gs_vec3 up)
+{
+    gs_vec3 f = gs_vec3_norm(gs_vec3_sub(target, position));
```

Comment:
remove subtraction. 
I think `target` should already be normalized? I'm not completely sure

Reply:
Looked around for camera rotation vectors used by `gs_camera_forward`. They are unit vectors!